src/views/ReadOnlyContactDetails.vue

```diff
@@ -0,0 +1,283 @@
+			profileEnabled,
+			contact: undefined,
+			localContact: undefined,
+		}
+	},
+
+	computed: {
+		/**
+		 * Read-only representation of the contact title and organization.
+		 *
+		 * @return {string}
+		 */
+		formattedSubtitle() {
+			const title = this.contact.title
+			const organization = this.contact.org
+
+			if (title && organization) {
+				return t('contacts', '{title} at {organization}', {
+					title,
+					organization,
+				})
+			} else if (title) {
+				return title
+			} else if (organization) {
+				return organization
+			}
+
+			return ''
+		},
+		addressbooks() {
+			return this.$store.getters.getAddressbooks
+		},
+		/**
+		 * Contact properties copied and sorted by rfcProps.fieldOrder
+		 *
+		 * @return {Array}
+		 */
+		sortedProperties() {
+			if (!this.localContact || !this.localContact.properties) {
+				return []
+			}
+			return this.localContact.properties
+				.toSorted((a, b) => {
+					const nameA = a.name.split('.').pop()
+					const nameB = b.name.split('.').pop()
+					return rfcProps.fieldOrder.indexOf(nameA) - rfcProps.fieldOrder.indexOf(nameB)
+				})
+		},
+
+		/**
+		 * Contact properties filtered and grouped by rfcProps.fieldOrder
+		 *
+		 * @return {object}
+		 */
+		groupedProperties() {
+			if (!this.sortedProperties) {
+				return {}
+			}
+			return this.sortedProperties.reduce((list, property) => {
+				if (!this.canDisplay(property)) {
+					return list
+				}
+				if (!list[property.name]) {
+					list[property.name] = []
+				}
+				list[property.name].push(property)
+				return list
+			}, {})
+		},
+		/**
+		 * The address book is read-only (e.g. shared with me).
+		 *
+		 * @return {boolean}
+		 */
+		addressbookIsReadOnly() {
+			return this.contact.addressbook?.readOnly
+		},
+		/**
+		 * Usable addressbook object linked to the local contact
+		 *
+		 * @return {string}
+		 */
+		addressbook() {
+			return this.contact.addressbook.id
+		},
+
+		/**
+		 * Fake model to use the propertyGroups component
+		 *
+		 * @return {object}
+		 */
+		groupsModel() {
+			return {
+				readableName: t('mail', 'Contact groups'),
+				icon: 'icon-contacts-dark',
+			}
+		},
+	},
+	watch: {
+		contact: {
+			handler(contact) {
+				this.updateLocalContact(contact)
+			},
+			immediate: true,
+		},
+	},
+	async beforeMount() {
+		// Init client and stores
+		await client.connect({ enableCardDAV: true })
+		const principalsStore = usePrincipalsStore()
+		principalsStore.setCurrentUserPrincipal(client)
+		await this.$store.dispatch('getAddressbooks')
+
+		// Fetch contact
+		await this.fetchContact()
+	},
+	methods: {
+		async fetchContact() {
+			const email = this.contactEmailAddress
+
+			const result = await Promise.all(
+				this.addressbooks.map(async (addressBook) => [
+					addressBook.dav,
+					await addressBook.dav.addressbookQuery([{
+						name: [NS.IETF_CARDDAV, 'prop-filter'],
+						attributes: [['name', 'EMAIL']],
+						children: [{
+							name: [NS.IETF_CALDAV, 'text-match'],
```

Comment:
```suggestion
							name: [NS.IETF_CARDDAV, 'text-match'],
```

To resolve https://github.com/nextcloud/contacts/pull/4194#issuecomment-2514808458